docs/source/blueprints/BP000.rst

```diff
@@ -0,0 +1,307 @@
+something.
+
+  * Create a common standard that will make it easier for readers.
+
+With an open RFC, authors tend to organize the ideas in different ways with
+different sections. Having a very well structured document with common sections
+will make it easier for readers to understand the problem and the solution.
+
+  * Track the progress of a significant implementation.
+
+We could use the blueprints header “status” line to track the progress of some
+features. We could even have a page parsing and listing all the blueprints with
+the title, author, status, and target milestone for that feature.
+
+  * Find the middle ground between an “overthinking” and “auto-pilot.”
+
+Last but not least: we are *not trying* to overthink here and/or slow down our
+development processes. The idea is to have a lightweight document, very
+objective that will save us time in the medium and long run. We don’t need to
+overthink trying to find any possible scenario that we don’t have a use case.
+But we should also avoid the “auto-pilot” mode to fix things.
+
+
+Specification
+*************
+
+One blueprint per topic
+-----------------------
+
+Try to follow the minimalist approach and be concise with content relevant to
+one particular topic. If you have a more general topic to discuss, you should
+set the type as “Epic Blueprint” (more below) but still try to be concise and
+focused on the subject.  File format and location
+
+Our current documentation already uses ReStructuredText (.rst) format, so we
+will adopt .rst format here too. All blueprints will be located inside
+`docs/source/blueprints` with the filename `BPXXX.rst`, where XXX is the number
+of the blueprint. Just pick the next number available for your blueprint.
+Write for your audience
+
+As mentioned before, your blueprint will be read by your peers, future members,
+and yourself. Keep in mind that your audience is developers with a minimal
+understanding of the Avocado internals, but be gentle and provide any context
+to understand the problem.  Blueprints types
+
+Currently, we have the blueprint types:
+
+ * Architectural Blueprint: Any blueprint changing or introducing a new core
+   feature or architectural change to Avocado.
+
+ * Process Blueprint: Any blueprint that is not implementing a new core
+   feature, but changing how the project works. This could be, for instance,
+   related to the repositories or processes.
+
+ * Meta Blueprint: A blueprint about blueprints. Like this one and any future
+   blueprint that changes our blueprints styles and methods.
+   
+ * Epic Blueprint: A blueprint that is touching in multiple areas and is too
+   big to have all the documentation in one single blueprint. We could split
+   epic blueprints into smaller blueprints or issues (if they are small and
+   easy to understand).
+
+ * Utility Blueprint: A blueprint with the intent to describe a new utility
+   module or a new plugin.
```

Comment:
I guess it could also be called (non-core and thus not architectural) "component" blueprint. Or "enhancement" blueprint. It could relate to additional plugins or components and not just utilities.

Reply:
Yes, I'm ok with Component.